packages/ui/src/utils.js

```diff
@@ -6,18 +6,29 @@ export function getStatsFromAttributes (bufficornAttributes) {
 }
 
 export function normalizedChartData (stats, bufficornsArray) {
-  const result = bufficornsArray.map(bufficorn => {
-    return calculateHightestNumber(bufficorn)
-  })
-  const finalResult = calculateHightestNumber(result)
-  return Object.entries(stats).reduce(
-    (acc, [key, value]) => ({ ...acc, [key]: value / finalResult }),
-    []
-  )
+  if (bufficornsArray) {
+    const result = bufficornsArray.map(bufficorn => {
+      return calculateHightestNumber(bufficorn)
+    })
+    const finalResult = calculateHightestNumber(result)
+    return Object.entries(stats).reduce(
+      (acc, [key, value]) => ({ ...acc, [key]: value / finalResult + 0.05 }),
+      []
+    )
+  } else {
+    return {
+      vigor: 0.1,
+      stamina: 0.1,
+      speed: 0.1,
+      agility: 0.1,
+      coat: 0.1,
+      coolness: 0.1
+    }
+  }
 }
 
 function calculateHightestNumber (list) {
-  return Math.max(...Object.values(list).filter(Number.isFinite))
+  return Math.max(...Object.values(list).filter(Number.isFinite)) - 0.05
```

Comment:
Can you add a comment explaining that you need to add `- 0.05` to improve the graphic UI?